ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -298,12 +301,24 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
         }
         let capabilities = capabilitiesStr.compactMap { Capability(rawValue: $0) }
 
+        var forwardJumpInterval: NSNumber = lastForwardJumpInterval;
+        if let explicitForwardJumpInterval = options["forwardJumpInterval"] as? NSNumber {
+            forwardJumpInterval = explicitForwardJumpInterval;
+            lastForwardJumpInterval = forwardJumpInterval;
+        }
```

Comment:
Would love to learn if there's a better way to accomplish this (e.g. to read `player.remoteComments[.skipForward[interval]]` directly, but I'm no Swift expert.

Reply:
@fivecar (not a swift expert either) it might be cleaner to simply have a `forwardJumpInterval` and use a "Nil Coalescing" operator to set it every time. Something like this:

```swift
private var forwardJumpInterval: NSNumber = 0;

...

public func update(....) {
    forwardJumpInterval = options["forwardJumpInterval"] as? NSNumber ?? forwardJumpInterval
    backwardJumpInterval = options["backwardJumpInterval"] as? NSNumber ?? backwardJumpInterval
    ...
}


```

Reply:
Awesome suggestion, and so much cleaner! Thank you, @jspizziri !

Reply:
@fivecar it looks like the nil coalescing operator got dropped in the cleanup. Was that intentional?

Reply:
Ack, no, not intentional. Thanks for catching, @jspizziri . Will re-up.